### DISCUSSION TITLE
General API access through Home Assistant?

### DISCUSSION THREAD

**Discussion by chrisharrington** [blakeblackshear/frigate#5591] [Ask A Question] [answered]
Hi there!

I'm interested in using the full Frigate API proxied through Home Assistant. Is that possible? I've seen that there are three notification API endpoints available, but I would love to be able to pull down a list of all events from Frigate through HA.

Thanks!
Chris

**Comment by NickM-27** [comment]
You can still access the full API using port 5000 on the docker container / addon

**Comment by chrisharrington** [comment, marked as answer]
I ended up just exposing Frigate to the web and accessed the Frigate HTTP API directly. Thanks!
